revolv/project/views.py

```diff
@@ -158,6 +159,31 @@ def success_url(self):
         return '/project/{0}'.format(self.kwargs.get('pk'))
 
 
+def validate_payment(request, *args, **kwargs):
```

Comment:
This seems like it could be a `FormView`? I'm down for not using class based views if we don't have to, but it seems like you're just implementing `form_valid` basically